doc/update065.md

```diff
@@ -21,3 +21,28 @@ tail -f ~/.komodo/VRSC/debug.log
 ```
 
 Make sure chain is running smoothly and notarisations are progressing.
+
+- Remove SFUSD.
+```bash
+# Return remaining SFUSD funds
+cd ~/sfusd-core/src
+./smartusd-cli sendtoaddress SVpijx2FxX67ny8LAtC4ogrvpSWzwkda5x $(./smartusd-cli getbalance) "" "" true
+
+# Stop SFUSD and delete source code
+./smartusd-cli stop
+cd ~/
+rm -rf ~/sfusd-core
+
+# Delete SFUSD user data
+rm -rf ~/.smartusd
```

Comment:
I think we should remove this. notaries can take care of their disk space when necessary. if someone is copy pasting commands from here blindly/sleepily, they might lost coins in change adresses etc., if somehow the `sendtoaddress` on line 29 failed and they didn't notice

Reply:
done